djangocms_version_locking/helpers.py

```diff
@@ -1,9 +1,10 @@
 from django.contrib import admin
 from django.contrib.contenttypes.models import ContentType
 
-from djangocms_version_locking.models import VersionLock
+from djangocms_versioning import constants
 
-from .admin import VersionLockAdminMixin
+from djangocms_version_locking.models import VersionLock
```

Comment:
Import order (admin goes before models). Could also turn it into a relative import